test/spec/test_case.zig

```diff
@@ -12,68 +12,46 @@ const capella = consensus_types.capella;
 const deneb = consensus_types.deneb;
 const electra = consensus_types.electra;
 
-// TODO: Most of the runners have fixed ssz types ie. Given a dir, we can already derive which
-// snappy file has what type and have this info written in test_type/schema.
-// There is an exception which is `transition`. One needs to read meta.yaml to figure out
-// whether a block_x.ssz_snappy belong to old fork or new fork. Current design of loadTestCase
-// does not handle this.
-pub fn loadTestCase(comptime Schema: type, comptime SchemaOut: type, dir: std.fs.Dir, allocator: std.mem.Allocator, out: *SchemaOut) !void {
-    var it = dir.iterate();
+pub const BlsSetting = enum {
+    default,
+    required,
+    ignored,
+};
 
-    while (try it.next()) |entry| {
-        // Ignore all non-file
-        if (entry.kind != .file) continue;
-        const name = entry.name;
-        // Ignore all hidden files
-        if (name.len == 0) continue;
-        if (name[0] == '.') continue;
+pub fn loadBlsSetting(allocator: std.mem.Allocator, dir: std.fs.Dir) BlsSetting {
+    var file = dir.openFile("meta.yaml", .{}) catch return .default;
```

Comment:
Would be a good idea to add `meta.yaml` with `bls_setting: 0` and let others change the value as they see fit, instead of having them creating that file